deafrica/monitoring/s2_gap_report.py

```diff
@@ -168,11 +169,21 @@ def generate_buckets_diff(
 )
 @update_stac
 @slack_url
+@click.option("--version", is_flag=True, default=False)
```

Comment:
This should be added to all CLI tools if it's going to be added to one.

Reply:
And if it is, it should be one of the imported ones.

Reply:
I'll do that